### PR TITLE
chore: run e2e tests on Cloud Run

### PR DESCRIPTION
**Pull request by aabmass**

### CODE REVIEW

File: e2e-test-server/src/index.ts [hunk 1]
```diff
@@ -102,13 +61,124 @@ async function pubSubPull(): Promise<void> {
...
+function pubSubPush() {
+  if (constants.PUSH_PORT === undefined) {
+    throw new Error('PUSH_PORT must be set when in push subscription mode');
+  }
+
+  const app = express();
+
+  app
+    .use(express.json())
+    .post('/', (req, res) => {
```
Comment (dashpole) [inline e2e-test-server/src/index.ts:77]: do we need to handle /ready and /alive as well?

Comment (aabmass) [reply e2e-test-server/src/index.ts:77]: Didn't know those are needed, what's it for?

Comment (dashpole) [reply e2e-test-server/src/index.ts:77]: Ah, might just be for app-engine.

Comment (dashpole) [reply e2e-test-server/src/index.ts:77]: We can add it for that later if needed.

**Review by dashpole — APPROVED** [review]

File: e2e-test-server/src/index.ts (hunk 1 above)
Comment (aabmass) [reply e2e-test-server/src/index.ts:77]: Ok, can add those when I add App Engine 👍

**Pull request merged**